README.Rmd

```diff
@@ -125,12 +125,33 @@ say('time')
 say("ain't that some shit", "chicken")
 ```
 
+Add some color: 
```

Comment:
pretty sure we need to load `crayon` for these examples

Reply:
I'll load `multicolor` and that should take care of both

Reply:
Thanks for pointing this out!